uiplib/GUI.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, settings):
         self.settings = settings
         # base window
         self.root = Tk()
-        self.root.resizable(width=False, height=False)
+        self.root.resizable(width=True, height=True)
```

Comment:
Yeah, just a cosmetic point of view, making sure that adding components to it doesn't mess it up much, since we're making it pixel perfect. :stuck_out_tongue: